lib/rspec/core/test_unit_assertions_adapter.rb

```diff
@@ -6,24 +6,9 @@ module Core
     module TestUnitAssertionsAdapter
       include ::Test::Unit::Assertions
 
-      # If using test/unit from Ruby core with Ruby 1.9+, it includes
-      # MiniTest::Assertions by default. Note the upcasing of 'Test'.
-      #
-      # If the test/unit gem is being loaded, it will not include any minitest
-      # assertions.
-      #
-      # Only if Minitest 5.x is included / loaded do we need to worry about
-      # adding a shim for the new updates. Thus instead of checking on the
-      # RUBY_VERSION we need to check ancestors.
-      begin
-        # MiniTest is 4.x.
-        # Minitest is 5.x.
-        if ancestors.include?(::Minitest::Assertions)
-          require 'rspec/core/minitest_assertions_adapter'
-          include ::RSpec::Core::MinitestAssertionsAdapter
-        end
-      rescue NameError
-        # No-op. Minitest 5.x was not loaded.
```

Comment:
I feel half this comment and code is still warranted? Otherwise `MiniTest 4.x` will break?

Reply:
MiniTest was removed from Ruby 2.2.
https://docs.ruby-lang.org/en/2.1.0/MiniTest.html - still present in 2.1, but not in 2.2:
https://docs.ruby-lang.org/en/2.2.0/MiniTest.html

Do you think we might break compatibility with `minitest` the gem 4.x? I'll double-check.

Reply:
> Do you think we might break compatibility with minitest the gem 4.x? 
Yes, silently, is what I am concerned about

Reply:
Ah I see, this is a refactor of the existing check, so the comment just needs adapting